src/Perlang.ConsoleApp/Program.cs

```diff
@@ -57,6 +57,7 @@ internal enum ExitCodes
         private readonly PerlangInterpreter interpreter;
         private readonly Action<string> standardOutputHandler;
         private readonly Action<string> standardErrorHandler;
+        private readonly HashSet<WarningType> disabledWarningsAsErrors = new();
```

Comment:
Note: this MR does not include any mechanism for actually populating this field. Will be handled by #188.